README.md

```diff
@@ -28,11 +28,11 @@ Here are a few things that were removed but did not have deprecation warnings in
 
 ## Automatic Node.js Polyfills Removed
 
-In the early days, webpack's aim was to allow running most node.js modules in the browser, but the module landscape changed and many module uses are now written mainly for frontend purposes. webpack <= 4 ships with polyfills for many of the node.js core modules, which are automatically applied once a module uses any of the core modules (i.e. the `crypto` module).
+In the early days, webpack's aimed to allow running most node.js modules in the browser, but the module landscape changed and many module uses are now written mainly for frontend purposes. webpack <= 4 ships with polyfills for many of the node.js core modules, which are automatically applied once a module uses any of the core modules (i.e. the `crypto` module).
```

Comment:
The current one is correct, however, if you want to suggest a change, then it could be `webpack was aimed...` instead of `webpack's aimed`

Reply:
If the current one is correct I see no reason to propose a change. 
I will change it back.
Thanks for the review.